Documentation for Partial and Downstream Pipelines in CI Visibility (#15726)
DataDog/documentation: content/en/continuous_integration/pipelines/buildkite.md

**Note**: The Pipelines page shows data for only the default branch of each repository.

### Partial and downstream pipelines

> In the `Pipeline Executions` page, you can now use the filters below in the search bar:

Reviewer: ```suggestion
In the **Pipeline Executions** page, you can use the filters below in the search bar:
```

> * `Downstream Pipeline` - possible values: `true`, `false` 
> * `Manually Triggered` - possible values: `true`, `false` 
> * `Partial Pipeline` - possible values: `retry`, `paused`, `resumed`

Reviewer: ```suggestion
`Downstream Pipeline`
: Possible values: `true`, `false`

`Manually Triggered`
: Possible values: `true`, `false`

`Partial Pipeline`
: Possible values: `retry`, `paused`, `resumed`
```

> {{< img src="ci/partial_retries_search_tags.png" alt="GitLab Partial Retry Filters" style="width:100%;">}}

Reviewer: ```suggestion
{{< img src="ci/partial_retries_search_tags.png" alt="The Pipeline executions page with Partial Pipeline:retry entered in the search query" style="width:100%;">}}
```

Reviewer: Just following up on this image:
- Is the Pipeline Executions page the correct place to enter this search? The UI of that page doesn't seem to match your screenshot. Perhaps this is a limitation of the demo account?
- I updated the alt text with our internal guidelines in mind

Author: Yes, the Pipeline Executions page is the correct place for the search, but the demo account does not contain a lot of our newest features. Updating our demo environment is something we are actively working on :)

> These filters can also be applied via the facet panel on the left hand side of the page.

Reviewer: ```suggestion
These filters can also be applied through the facet panel on the left hand side of the page.
```

> {{< img src="ci/partial_retries_facet_panel.png" alt="GitLab Partial Retry Facet Panel" style="width:40%;">}}

Reviewer: ```suggestion
{{< img src="ci/partial_retries_facet_panel.png" alt="The facet panel with Partial Pipeline facet expanded and the value Retry selected, the Partial Retry facet expanded and the value true selected" style="width:40%;">}}
```

**Note**: The Pipelines page shows data for only the default branch of each repository.

> ### Partial and downstream pipelines
> In the **Pipeline Executions** page, you can use the filters below in the search bar:

Reviewer: ```suggestion
### Partial and downstream pipelines

In the **Pipeline Executions** page, you can use the filters below in the search bar:

```
